scene:
  env_spacing: 30
  rows: 2
  cols: 2
  joint_names:
    - FL1_
    - FL2_
    - FL3_
    - FR1_
    - FR2_
    - FR3_
    - RL1_
    - RL2_
    - RL3_
    - RR1_
    - RR2_
    - RR3_
env_config:
  sim_dt: 0.001
  decimation: 10
  max_episode_length: 1000
  episode_length_s: 10

train:
  num_envs: 8192
  num_iterations: 1200
  rows: 10
  cols: 10
  total_timesteps: 98e6
  n_steps: 128
  batch_size: 8192
  learning_rate: 3.5e-4
  clip_range: 0.2
  gamma: 0.992
  ent_coef: 0.0025

eval:
  num_envs: 30
  rows: 1
  cols: 1

save_config:
  save_path: ../../models/isaac_models/
  save_prefix: ruu_ppo_model
